solidity/ecdsa/deploy/00_resolve_random_beacon.ts

```diff
@@ -10,12 +10,19 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const RandomBeacon = await deployments.getOrNull("RandomBeacon")
 
+  const isRandomBeaconNeeded = function () {
+    if (hre.network.tags.chaosnet) {
```

Comment:
Where is this tag set and in what circumstances?

Reply:
I thought it would be passed during deployment.
But I can add it permanently to [mainnet configuration in hardhat.config.ts](https://github.com/keep-network/keep-core/blob/a0817e53bc6ce1c85013ee254d24b88cc78b881a/solidity/ecdsa/hardhat.config.ts#L129).

Reply:
In the end `chasonet` was added to `development` and `goerli` networks.